rskj-core/src/test/java/co/rsk/peg/federation/FederationSupportImplTest.java

```diff
@@ -2282,6 +2282,28 @@ void voteFederationChange_create_returnsSuccessfulResponseCode() {
 
         }
 
+        @Test
+        void voteFederationChange_whenCreateIsCalledTwiceBySameAuthorizer_returnsGenericErrorResponseCode() {
+
+            // Arrange
+
+            Transaction tx = TransactionUtils.getTransactionFromCaller(signatureCache, FederationChangeCaller.FIRST_AUTHORIZED.getRskAddress());
+            ABICallSpec abiCallSpec = new ABICallSpec(FederationChangeFunction.CREATE.getKey(), new byte[][]{});
+
+            // Act
+
+            // First create call
+            federationSupport.voteFederationChange(tx, abiCallSpec, signatureCache, bridgeEventLogger);
```

Comment:
We should assert the first call was successful

Reply:
I was following previous suggestions, about avoiding to test what has been already tested. The test exactly right before this up to this point is exactly the same and tested this. But it won't hurt to assert it.

Reply:
Updated.